apps/edxec/vars/all/main.yml

```diff
@@ -0,0 +1,35 @@
+# Application default configuration
+
+# -- routes
+edxec_host: "edxec.{{ project_name}}.{{ domain_name }}"
+
+# -- edxec
+edxec_image_name: "fundocker/edxec"
+edxec_image_tag: "master"
+edxec_django_port: "8000"
+edxec_app_replicas: 1
+edxec_secret_name: "edxec-{{ edxec_vault_checksum | default('undefined_edxec_vault_checksum') }}"
+edxec_activate_http_basic_auth: false
+edxec_yq_image_name: "mikefarah/yq"
```

Comment:
do you really think it is needed to set this image name in a variable ?

Reply:
:man_shrugging: I wanted the tag to be configurable, so by extension, the image is also configurable. Do you think it's useless?

Reply:
I think yes but it's not a big deal